services/policies/README.md

```diff
@@ -0,0 +1 @@
+# Policies Service
```

Comment:
Please add something. If you just copy the description from the changelog we will already have a good starting point.